src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.tsx

```diff
@@ -1,34 +1,25 @@
 import { Box, Serif } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
 import { SystemContextConsumer } from "Artsy"
-import { track } from "Artsy/Analytics"
-import * as Schema from "Artsy/Analytics/Schema"
 import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
 
 import { ArtworkSidebarAuctionPartnerInfo_artwork } from "__generated__/ArtworkSidebarAuctionPartnerInfo_artwork.graphql"
 
 export interface ArtworkSidebarAuctionPartnerInfoProps {
   artwork: ArtworkSidebarAuctionPartnerInfo_artwork
 }
 
-@track()
 export class ArtworkSidebarAuctionPartnerInfo extends React.Component<
   ArtworkSidebarAuctionPartnerInfoProps
 > {
-  @track(() => ({
```

Comment:
This call was previously decorating the handler for clicking the BP link, but when that link moved it was suddenly decorating the render function.